src/main/java/org/intellij/sdk/codesync/database/UserTable.java

```diff
@@ -0,0 +1,87 @@
+package org.intellij.sdk.codesync.database;
+
+import org.intellij.sdk.codesync.Database;
+import org.intellij.sdk.codesync.models.UserAccount;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class UserTable {
+
+    static String email;
+    static String access_token;
+    static String secret_key;
+    static String access_key;
+    static int is_active;
+
+    public static void insertNewUser(UserAccount userAccount){
+        setValues(userAccount);
+        String query = String.format("INSERT INTO user VALUES(%s, %s, %s, %s, %d)", email, access_token, secret_key, access_key, is_active);
+        Database.executeUpdate(query);
+    }
+
+    public static void updateUser(UserAccount userAccount){
+        setValues(userAccount);
+        String query = String.format("UPDATE user SET access_token = %s, secret_key = %s, access_key = %s, is_active = %d WHERE email = %s", access_token, secret_key, access_key, is_active, email);
+        Database.executeUpdate(query);
+    }
+
+    public static void updateAllUsersInActive(){
+        String query = "UPDATE user SET is_active = 0";
+        Database.executeUpdate(query);
+    }
+
+    public static UserAccount getActiveUser(){
+        String query = "SELECT * FROM user WHERE is_active = 1";
```

Comment:
Can we move inline queries to a separate file? Here is an exmaple: https://github.com/codesyncapp/codesyncserver/blob/master/queries/users/users.py#L8